pkg/apply/prune/prune.go

```diff
@@ -97,10 +97,11 @@ func (po *PruneOptions) Prune(pruneObjs []*unstructured.Unstructured,
 		klog.V(5).Infof("attempting prune: %s", pruneID)
 		// Check filters to see if we're prevented from pruning/deleting object.
 		var filtered bool
+		var reason string
```

Comment:
Nit: Why is `reason` defined outside of the loop? It seems like it is only used inside.